pom.xml

```diff
@@ -38,7 +38,7 @@
     </scm>
 
     <properties>
-        <es.version>7.9.3</es.version>
+        <es.version>7.0.1</es.version>
```

Comment:
Reverted the upgrade here: https://github.com/confluentinc/kafka-connect-elasticsearch/pull/557/files

It looks like the MappingMetaData change does not need to be reverted